Tools/TestWebKitAPI/Tests/WebKitGLib/TestInputMethodContext.cpp

```diff
@@ -108,12 +111,30 @@ static gboolean webkitInputMethodContextMockFilterKeyEvent(WebKitInputMethodCont
     bool isControl = state & GDK_CONTROL_MASK;
     bool isShift = state & GDK_SHIFT_MASK;
 #elif PLATFORM(WPE)
-    struct wpe_input_keyboard_event* wpeKeyEvent = static_cast<struct wpe_input_keyboard_event*>(keyEvent);
-    uint32_t keyval = wpeKeyEvent->key_code;
-    bool isKeyPress = wpeKeyEvent->pressed;
-    gunichar character = wpe_key_code_to_unicode(keyval);
-    bool isControl = wpeKeyEvent->modifiers & wpe_input_keyboard_modifier_control;
-    bool isShift = wpeKeyEvent->modifiers & wpe_input_keyboard_modifier_shift;
+    uint32_t keyval;
```

Comment:
Quite ugly method, gettin' even more ugly. This could need some love & refactoring -- but out of scope for this PR.